hil/test_common.py

```diff
@@ -329,6 +332,48 @@ def site_layout():
                                  node['name'], nic['name'])
 
 
+def obmd_cfg():
+        """Fixture launching obmd with a config.
+
+        The fixture yields the config as a parsed json object. The obmd
+        process is teminated when the test completes.
+
+        Like many of the functions in this module, this is intended to
+        be used as a fixture, but not declared here as such; individual
+        modules should declare it as a fixture.
+        """
+        portno = 8833
+        cfg = {
+            'ListenAddr': ":" + str(portno),
+            'AdminToken': "8fcfe5d3f8ca8c4b87d0f8ae86b43bca",
+            'DBType': 'sqlite3',
+            'DBPath': ':memory:',
+        }
+
+        (fd, name) = tempfile.mkstemp()
+        os.write(fd, json.dumps(cfg))
+        os.close(fd)
+        obmd_proc = subprocess.Popen(['obmd', '-config', name])
```

Comment:
suggestion: might be a good idea to check for exceptions here, will make it easier to debug if we are unable to install obmd or something else goes wrong later. I think we should be checking for OSError.